Yona/Yona.xcodeproj/project.pbxproj

```diff
@@ -66,7 +66,7 @@
 		166CB84A1D2C5688008B425D /* MeWeekDetailWeekViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 166CB8491D2C5688008B425D /* MeWeekDetailWeekViewController.swift */; };
 		166CB84D1D2C5E58008B425D /* YonaButtonsTableHeaderView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 166CB84B1D2C5E58008B425D /* YonaButtonsTableHeaderView.swift */; };
 		166CB84E1D2C5E58008B425D /* YonaButtonsTableHeaderView.xib in Resources */ = {isa = PBXBuildFile; fileRef = 166CB84C1D2C5E58008B425D /* YonaButtonsTableHeaderView.xib */; };
-		166CB8501D2E640A008B425D /* FriendsDayViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 166CB84F1D2E640A008B425D /* FriendsDayViewController.swift */; };
+		166CB8501D2E640A008B425D /* FriendsDashBoardViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 166CB84F1D2E640A008B425D /* FriendsDashBoardViewController.swift */; };
```

Comment:
In English, 'dashboard' is a single word, so don't capitalize the `B`

Reply:
Done.